src/settings/routes/ProxyServerSettingsRoute/ProxyServerSettingsRoute.test.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types';
+import { renderWithIntl } from '@folio/stripes-erm-testing';
+import { MemoryRouter } from 'react-router-dom';
+
+import { useCallout } from '@folio/stripes/core';
+import { Button } from '@folio/stripes/components';
+import { Button as ButtonInteractor } from '@folio/stripes-testing';
+
+import translationsProperties from '../../../../test/helpers/translationsProperties';
+import ProxyServerSettingsRoute from './ProxyServerSettingsRoute';
+
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useCallout: jest.fn().mockReturnValue({
+    sendCallout: jest.fn()
+  }),
+  useOkapiKy:jest.fn()
+}));
+
+const SaveButton = (props) => {
+  return <Button onClick={props.onSave}>SaveButton</Button>;
+};
+
+const DeleteButton = (props) => {
+  return <Button onClick={props.onDelete}>DeleteButton</Button>;
+};
+
+SaveButton.propTypes = {
+  onSave: PropTypes.func,
+};
+
+DeleteButton.propTypes = {
+  onDelete: PropTypes.func,
+};
+
+jest.mock('../../components/ProxyServerSettingsConfig/ProxyServerSettingsForm', () => {
+  return (props) => (
+    <div>
+      <div>ProxyServerSettingsForm</div>
+      <SaveButton {...props} />
+      <DeleteButton {...props} />
+    </div>
+  );
+});
+
+const initialValues = {
+  'stringTemplates': []
+};
+
+describe('ProxyServerSettingsRoute', () => {
+  describe('rendering the route', () => {
+    let renderComponent;
+    beforeEach(() => {
+      jest.clearAllMocks();
+      renderComponent = renderWithIntl(
+        <MemoryRouter>
+          <ProxyServerSettingsRoute
+            initialValues={initialValues}
+          />
+        </MemoryRouter>,
+        translationsProperties
+      );
+    });
+
+    test('renders the ProxyServerSettingsForm component', () => {
+      const { getByText } = renderComponent;
+      expect(getByText('ProxyServerSettingsForm')).toBeInTheDocument();
+    });
+
+    test('clicking on the SaveButton fires the callout', async () => {
+      await ButtonInteractor('SaveButton').click();
+      expect(useCallout).toHaveBeenCalled();
```

Comment:
Can this test be updated to check the callout itself, similar to what's happening in this PR please? :)

https://github.com/folio-org/ui-local-kb-admin/pull/307#pullrequestreview-1244130215

Reply:
Ditto for the below test as well